modules/ROOT/pages/introduction-to-tiny-comments.adoc

```diff
@@ -37,6 +37,20 @@ The Comments plugin allows the user to perform the following functions:
 * Resolve a comment thread
 * Delete a comment or comment thread
 
+=== Supported content
```

Comment:
Does this need to be in the `annotations.adoc` page as well? If so we should make this a partial and include it in both places.

Reply:
I have split out the supported content into its own file and added it to the annotation and comments pages.